pages/reference/cli.md

```diff
@@ -81,76 +113,76 @@ environment variable (in the same standard URL format).
+	- [key &#60;id&#62;](#key-id)
+	- [key rm &#60;id&#62;](#key-rm-id)
+	- [key add &#60;name&#62; [path]](#key-add-name--path)
 
 - Logs
 
-	- [logs &#60;uuid&#62;](#logs-60-uuid-62-)
+	- [logs &#60;uuid&#62;](#logs-uuid)
 
 - Sync
 
-	- [sync [uuid]](#sync-uuid-)
+	- [sync [uuid]](#sync-uuid)
 
 - SSH
 
-	- [ssh [uuid]](#ssh-uuid-)
+	- [ssh [uuid]](#ssh-uuid)
 
 - Notes
 
-	- [note &#60;|note&#62;](#note-60-note-62-)
+	- [note &#60;|note&#62;](#note-note)
 
 - OS
 
-	- [os versions &#60;type&#62;](#os-versions-60-type-62-)
-	- [os download &#60;type&#62;](#os-download-60-type-62-)
-	- [os build-config &#60;image&#62; &#60;device-type&#62;](#os-build-config-60-image-62-60-device-type-62-)
-	- [os configure &#60;image&#62; [uuid] [deviceApiKey]](#os-configure-60-image-62-uuid-deviceapikey-)
-	- [os initialize &#60;image&#62;](#os-initialize-60-image-62-)
+	- [os versions &#60;type&#62;](#os-versions-type)
+	- [os download &#60;type&#62;](#os-download-type)
+	- [os build-config &#60;image&#62; &#60;device-type&#62;](#os-build-config-image--device-type)
+	- [os configure &#60;image&#62; [uuid] [deviceApiKey]](#os-configure-image--uuid--deviceapikey)
+	- [os initialize &#60;image&#62;](#os-initialize-image)
 
 - Config
 
 	- [config read](#config-read)
-	- [config write &#60;key&#62; &#60;value&#62;](#config-write-60-key-62-60-value-62-)
-	- [config inject &#60;file&#62;](#config-inject-60-file-62-)
+	- [config write &#60;key&#62; &#60;value&#62;](#config-write-key--value)
+	- [config inject &#60;file&#62;](#config-inject-file)
 	- [config reconfigure](#config-reconfigure)
 	- [config generate](#config-generate)
 
 - Preload
 
-	- [preload &#60;image&#62;](#preload-60-image-62-)
+	- [preload &#60;image&#62;](#preload-image)
 
 - Settings
 
 	- [settings](#settings)
 
 - Wizard
 
-	- [quickstart [name]](#quickstart-name-)
+	- [quickstart [name]](#quickstart-name)
 
 - Local
 
-	- [local configure &#60;target&#62;](#local-configure-60-target-62-)
-	- [local flash &#60;image&#62;](#local-flash-60-image-62-)
-	- [local logs [deviceIp]](#local-logs-deviceip-)
+	- [local configure &#60;target&#62;](#local-configure-target)
+	- [local flash &#60;image&#62;](#local-flash-image)
+	- [local logs [deviceIp]](#local-logs-deviceip)
 	- [local scan](#local-scan)
-	- [local ssh [deviceIp]](#local-ssh-deviceip-)
-	- [local push [deviceIp]](#local-push-deviceip-)
-	- [local stop [deviceIp]](#local-stop-deviceip-)
+	- [local ssh [deviceIp]](#local-ssh-deviceip)
+	- [local push [deviceIp]](#local-push-deviceip)
+	- [local stop [deviceIp]](#local-stop-deviceip)
 
 - Deploy
 
-	- [build [source]](#build-source-)
-	- [deploy &#60;appName&#62; [image]](#deploy-60-appname-62-image-)
+	- [build [source]](#build-source)
+	- [deploy &#60;appName&#62; [image]](#deploy-appname--image)
 
 - Utilities
 
 	- [util available-drives](#util-available-drives)
 
 # Application
 
-## app create &#60;name&#62;
+## <a name="#app-create-name"></a>app create &#60;name&#62;
```

Comment:
Looks like this is built from the previous CLI docs, not the latest, is that right?

Reply:
@pimterry yes, I'll rebase this with the latest changes.